ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -35,6 +34,7 @@ OSKErrorCode OnScreenKeyboard::launch(OSKConfig oskConfig) {
   std::scoped_lock lock(oskLaunchExitCtrlMutex);
 
   oskHandle.oskState_=OSK_STATE_LAUNCH_INPROGRESS;
+   NotificationCenter::subWindowCenter().emit("KeyboardEventNotification",keyboardWillShow);
```

Comment:
Need to discuss on this KeyboardEventNotification event emit here .
If this is the right place to invoke. Since this will happen from main/input thread synchronously.

Reply:
As part of improvent in notificationcenter user story. 
We are going to create the worker thread. With this issue can be resolved.